### ISSUE TITLE
Установка кодировки

### ISSUE THREAD

**Issue by visavi** [olamedia/nokogiri#23]
Подскажите как можно установить кодировку документа самостоятельно
если в html не указана кодировка, то nokogiri неверно отображает слова
к примеру при парсинге страницы https://games.mail.ru/pc/news/  срабатывает метод loadHtml  хотя должен сработать loadHtmlNoCharset  так как там в коде нет установленной кодировки

из-за этого весь текст отображается в неверной кодировке

если извернуться и сделать что-то типа такого
```
$html = file_get_contents('https://games.mail.ru/pc/news/');
$html .= '<meta http-equiv=Content-Type content="text/html;charset=UTF-8">';
```

то страница парсится в верной кодировке

**Comment by olamedia** [comment]
документ необходимо самостоятельно преобразовать в utf-8. вставка мета-тега кодировку не изменит и излишня, так как utf-8 и так используется.
воспользуйтесь функциями iconv / mbstring

**Comment by olamedia** [comment]
да, `nokogiri::fromHtmlNoCharset` это верный вариант в случае если просто нет указания кодировки.
`loadHtmlNoCharset` делает то же самое с уже созданным объектом `$nokogiri->loadHtmlNoCharset`

**Comment by morsvox** [comment]
Помогло:)спасибо
